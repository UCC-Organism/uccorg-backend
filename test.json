{
  "apiserver": {
    "port": 8080,
    "cachefile": "sample-data.json"
  },
  "test": {
    "outfile": "test.out",
    "startDate": "2013-09-20T06:20:00",
    "endDate": "2013-09-20T18:20:00",
    "xTime": 3000
  }
}
