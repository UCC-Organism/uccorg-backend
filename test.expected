"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
"event"
